### ISSUE TITLE
CLI panics if schema contains an json_document field type

### ISSUE THREAD

**Issue by xrl** [quickwit-oss/tantivy-cli#78]
I am indexing chemical structures, my basic schema:

```
{
  "index_settings": {
    "docstore_compression": "lz4"
  },
  "segments": [
    {
      "segment_id": "ecf472f4-5e12-42f4-989e-d2571f8ab745",
      "max_doc": 10,
      "deletes": null
    }
  ],
  "schema": [
    {
      "name": "smile",
      "type": "text",
      "options": {
        "indexing": {
          "record": "position",
          "fieldnorms": true,
          "tokenizer": "default"
        },
        "stored": true
      }
    },
    {
      "name": "description",
      "type": "json_object",
      "options": {
        "stored": true,
        "indexing": {
          "record": "position",
          "fieldnorms": true,
          "tokenizer": "default"
        }
      }
    }
  ],
  "opstamp": 11
}

```

and when I try to get the cli to run a query against this index:

```
% RUST_BACKTRACE=true tantivy serve --index=tmp/index 
thread 'main' panicked at 'unhandled type', /Users/xlange/.cargo/registry/src/github.com-1ecc6299db9ec823/tantivy-0.15.3/src/schema/field_entry.rs:233:38
stack backtrace:
   0: std::panicking::begin_panic
   1: <<tantivy::schema::field_entry::FieldEntry as serde::de::Deserialize>::deserialize::FieldEntryVisitor as serde::de::Visitor>::visit_map
   2: <tantivy::schema::schema::Schema as serde::de::Deserialize>::deserialize
   3: tantivy::core::index::load_metas
   4: tantivy::core::index::Index::open
   5: tantivy::commands::serve::run_serve_cli
   6: tantivy::main
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
```

**Comment by fulmicoton** [comment]
`json_object` are not supported yet. 
The issue tracking their support is #1234

This is not a duplicate though. The (terrible) bug is that we should not panic if the docmapper is unknown.

**Comment by fulmicoton** [comment]
Oops... Forget my previous message. I hadn''t spotted this was in tantivy-cli and not quickwit.

**Comment by xrl** [comment]
Looks like this has been fixed but it's not part of any tagged release

**Comment by xrl** [comment]
Fixed by #77
